dhcpcd-hooks/60-kano-dhcp-debug

```diff
@@ -0,0 +1,12 @@
+#
+# 60-kano-dhcp-debug
+#
+# Copyright (C) 2017 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License v2
+#
+# Simple script to diagnose dhcpcd events.
+# To enable, edit /etc/systemd/journalctl.conf and set Storage=persistent
+#
+
+ipaddr=`ip addr show wlan0 | grep inet | awk '{print $2}' | sed -s 's/\/.*//')`
+logger "kano-dhcp-debug: $reason ifup? $if_up interface? $interface carrier? $ifcarrier wireless? $ifwireless ssid=$ifssid ipaddr=$ipaddr"
```

Comment:
Will this not execute and do nothing regardless of whether the the logging is enabled? Is there any way to fully disable this when the logging isn't enabled?

Reply:
That is true, and I think I'll ship this file under `/usr/share` instead.